apps/sms/views/new_message/style/recipients.css

```diff
@@ -138,61 +142,39 @@ Do not remove.
   display: none;
 }
 
-/* Disable padding and give negative margin-end on empty contenteditable
+/**
+ * Disable padding and give negative margin-end on empty contenteditable
  * recipients when they do not have focus. This allows the Recipients List
  * to intelligently collapse when the final row contains no "assimilated"
  * (i.e. non-contenteditable) recipients. */
 .recipient[contenteditable=true]:empty {
-  /**
-   * We don't use -moz-margin-end here because of the bug 1103011, as currently
-   * directionality of empty contenteditable isn't inherited from parent, but
-   * always LTR, so -moz-margin-end equals to margin-right in both LTR and RTL
-   * modes. So to align it correctly we should override margin depending on text
-   * direction of parent element manually.
-   */
-  margin-right: -0.5rem;
   padding: 0;
 }
-
-.recipient[contenteditable=true]:focus,
-.recipient[contenteditable=true]:hover,
-.recipient[contenteditable=true]:empty:focus {
-  -moz-margin-end: 0.5rem;
-  -moz-padding-end: 0.3rem;
+/**
+ * We don't use margin-inline-end here because of the bug 1103011, as currently
+ * directionality of empty contenteditable isn't inherited from parent, but
+ * always LTR, so margin-inline-end equals to margin-right in both LTR and RTL
+ * modes. So to align it correctly we should override margin depending on text
+ * direction of parent element manually.
+ */
+html[dir="ltr"] .recipient[contenteditable=true]:empty {
+  margin-right: -0.5rem;
 }
-
-/* RTL for recipient panel */
-:-moz-dir(rtl) > .recipient[contenteditable] {
-  margin-left: 0.6rem;
-  margin-right: 0;
-
-  /**
-   * We need "text-align: right" only for the cases when currently editable
-   * recipient looses focus but not yet assimilated: case №1 - when we tap on
-   * thread-container - maybe fixed soon, case №2 - when we use screen reader,
-   * eg. single touch on message input steals focus from currently editable
-   * recipient, but doesn't yet give it to the message input. As the result
-   * flex-grow (see workaround for bug 1103011) returns and because of dir=auto
-   * LTR\RTL name would be always left\right aligned in both LTR and RTL modes.
```

Comment:
the screen reader is too crashy so I can't check this currently... I guess we can leave it for now and come back once we can check it.